src/containers/PersonContainer/CoverImage.tsx

```diff
@@ -61,25 +68,34 @@ const CoverImage: FC<CoverImageProps> = ({
   seatPictureSrc,
   electoralArea,
 }: CoverImageProps) => {
-  if (!personPictureSrc && !seatPictureSrc) {
-    return null;
-  }
+  const [personPictureSrcIsLoading, setPersonPictureSrcIsLoading] = useState(true);
+  const [seatPictureSrcIsLoading, setSeatPictureSrcIsLoading] = useState(true);
+
+  useEffect(() => {
+    if (personPictureSrc.data === null) {
+      // the visibility of the seat picture is dependent on the visiblity of the person picture
+      // if there's no person picture (it's null), allow the seat picture to be visible
+      setPersonPictureSrcIsLoading(false);
+    }
+  }, [personPictureSrc.data]);
 
   return (
     <Images>
-      <img
-        className="mzp-c-card-image"
-        src={
-          seatPictureSrc ||
-          EXAMPLE_COVER_VIEWS[Math.floor(Math.random() * EXAMPLE_COVER_VIEWS.length)]
-        }
-        alt={`${electoralArea || ""}`}
-      />
-      {personPictureSrc && (
+      <PlaceholderWrapper contentIsLoading={seatPictureSrcIsLoading || personPictureSrcIsLoading}>
+        <Seat
+          className="mzp-c-card-image"
+          src={seatPictureSrc.data === null ? EXAMPLE_COVER_VIEW : seatPictureSrc.data}
+          alt={`${electoralArea || ""}`}
+          onLoad={() => setSeatPictureSrcIsLoading(false)}
+        />
+      </PlaceholderWrapper>
```

Comment:
I see this placeholder but I also see you are using a fetch data container instead of the `LazyFetchDataContainer` for these values. Is the placeholder worth it then?

Reply:
`FetchDataContainer` is for containers that require lots of data from different DB models like EventContainer, PersonContainer, while `LazyFetchDataContainer` is for containers that require data from one or two DB models, like the components for votes of a person, or legislation sponsored by a person.

The `Placeholder` and the `LazyFetchDataContainer` are on the same level -- for use with smaller components. They are both needed. The placeholder is used to render a rectangular skeleton (in place of the actual pictures) so that the person name, contact, terms don't change positions when the pictures are finally loaded. While `LazyFetchDataContainer` is meant components that are initially hidden (no need to show rectangular skeleton).

Reply:
Makes sense to me! Thanks!